yarn.lock

```diff
@@ -4973,6 +5044,11 @@ emoji-regex@^8.0.0:
   resolved "https://registry.yarnpkg.com/emoji-regex/-/emoji-regex-8.0.0.tgz#e818fd69ce5ccfcb404594f842963bf53164cc37"
   integrity sha512-MSjYzcWNOA0ewAHpz0MxpYFvwg6yjy1NG3xteoqz644VCo/RPgnr1/GGt+ic3iJTzQ8Eu3TdM14SawnVUmGE6A==
 
+emojis-list@^2.0.0:
```

Comment:
whats that?

Reply:
that's probably Atila's making a mess. Thanks for catching it

Reply:
@DonGideon there is a number of updates that probably happened when I fixed some conflicts. Build is still passing and this package all packages with changes listed here, so it may be better to leave it as is and roll with the side effect